tests/db_functions/json/test_json_object.py

```diff
@@ -0,0 +1,31 @@
+from django.db.models.functions import JSONObject
+from django.test import TestCase
+
+from ..models import Author
+
+
+class JSONObjectTests(TestCase):
+    def setUp(self):
+        Author.objects.create(name='Ivan Ivanov', alias='iivanov')
+
+    def test_empty(self):
+        obj = Author.objects.annotate(json_object=JSONObject()).first()
+        self.assertDictEqual(obj.json_object, {})
+
+    def test_one(self):
+        obj = Author.objects.annotate(json_object=JSONObject(name='name')).first()
+        self.assertDictEqual(obj.json_object, {'name': 'Ivan Ivanov'})
+
+    def test_fully(self):
+        obj = Author.objects.annotate(json_object=JSONObject(
+            name='name',
+            alias='alias',
+            goes_by='goes_by',
+            age='age',
+        )).first()
+        self.assertDictEqual(obj.json_object, {
+            'name': 'Ivan Ivanov',
+            'alias': 'iivanov',
+            'goes_by': None,
+            'age': 30,
+        })
```

Comment:
We should test complex expression support as well

```suggestion
        obj = Author.objects.annotate(json_object=JSONObject(
            name=Lower('name'),
            alias='alias',
            goes_by='goes_by',
            age=F('age') * 2,
        )).first()
        self.assertEqual(obj.json_object, {
            'name': 'ivan ivanov',
            'alias': 'iivanov',
            'goes_by': None,
            'age': 60,
        })
```